__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -517,6 +522,57 @@ export const LongSections: StoryComponentType = {
     },
 };
 
+/**
+ * This is an example of an Accordion with a dropdown within each section.
+ * This demonstrates how the accordion keyboard interactions do not interfere
+ * with the dropdown's keyboard interactions.
+ */
+const items = [
+    <OptionItem label="Banana" value="banana" key={0} />,
+    <OptionItem label="Strawberry" value="strawberry" disabled key={1} />,
+    <OptionItem label="Pear" value="pear" key={2} />,
+];
+export const WithDropdown: StoryComponentType = {
+    render: function Render() {
+        const [value, setValue] = React.useState<any>(null);
+        const [singleOpened, setSingleOpened] = React.useState(false);
+
+        const [values, setValues] = React.useState<any>([]);
+        const [multiOpened, setMultiOpened] = React.useState(false);
+
+        return (
+            <Accordion animated={true}>
+                <AccordionSection header={`Single Select`}>
+                    {/* Adding height because overflow hidden in sections. */}
+                    <View style={singleOpened && {height: 200}}>
+                        <SingleSelect
+                            placeholder="Select an option"
+                            selectedValue={value}
+                            onChange={setValue}
+                            opened={singleOpened}
+                            onToggle={setSingleOpened}
+                        >
+                            {items}
+                        </SingleSelect>
+                    </View>
+                </AccordionSection>
+                <AccordionSection header={`Multi Select`}>
+                    <View style={singleOpened && {height: 200}}>
+                        <MultiSelect
+                            selectedValues={values}
+                            onChange={setValues}
+                            opened={multiOpened}
+                            onToggle={setMultiOpened}
+                        >
+                            {items}
+                        </MultiSelect>
```

Comment:
**question:** Is it expected that this `MultiSelect` instance should open on click/Enter/Space? I was trying this out in SB and the SingleSelect works as expected, but MultiSelect never opens.

For reference, I'm using this instance: https://5e1bf4b385e3fb0020b7073c-doqpquuorc.chromatic.com/?path=/story/accordion-accordion--with-dropdown

Reply:
@jandrade, in that version, it's cut off by the boundaries of the AccordionSection, which has `overflow: hidden`. I pushed a change to extend its height in [fd8dd7e](https://github.com/Khan/wonder-blocks/pull/2157/commits/fd8dd7edff68192b9fcca688b51a6bc1dae652cf), but this seems to be an old version.

Reply:
Gotcha, I now see it working! thanks for the context.